src/Sylius/Behat/Context/Api/Shop/CartContext.php

```diff
@@ -631,9 +631,9 @@ private function geOrderItemIdForProductInCart(ProductInterface $product, string
 
     private function changeQuantityOfOrderItem(string $orderItemId, int $quantity, string $tokenValue): void
     {
-        $request = Request::customItemAction('shop', 'orders', $tokenValue, HttpRequest::METHOD_PATCH, 'change-quantity');
+        $request = Request::customItemAction('shop', 'orders', $tokenValue, HttpRequest::METHOD_PATCH, sprintf('items/%s', $orderItemId));
 
-        $request->updateContent(['orderItemId' => $orderItemId, 'newQuantity' => $quantity]);
+        $request->updateContent(['newQuantity' => $quantity]);
```

Comment:
as we are changing this endpoint, lets rename it too 
```suggestion
        $request->updateContent(['quantity' => $quantity]);
```